File: Toph/String/Problem in the Printer/code.py
while True:
    s = input()
    if s == "the end.":
        break
    m = ""
    for char in s:
        if char == 'b':
            m += '6'
        elif char == 'g':
            m += '9'
        elif char == 'l':
            m += '1'
        elif char == 'o':
            m += '0'
        elif char == 's':
            m += '5'
        elif char == 'z':
            m += '2'
        elif char == '6':
            m += 'b'
        elif char == '9':
            m += 'g'
        elif char == '1':
            m += 'l'
        elif char == '0':
            m += 'o'
        elif char == '5':
            m += 's'
        elif char == '2':
            m += 'z'
        else:
            m += char
    print(m)

File: Problem Solving/Calculate nCr binomial coefficient for n & r/code.cpp
#include <bits/stdc++.h>
using namespace std;

#define fastio ios_base::sync_with_stdio(false); cin.tie(nullptr); cout.tie(nullptr)
#define endl '\n'

long long factorial(int k) {
    long long fact = 1;

    for (int i = 2; i <= k; ++i) {
        fact *= i;
    }

    return fact;
}

int main() {
    fastio;

    int n, r;
    cin >> n >> r;

    if (r > n || n < 0 || r < 0) {
        cout << "Invalid input" << endl;
        return 0;
    }

    long long fact_n = factorial(n);
    long long fact_r = factorial(r);
    long long fact_nmr = factorial(n - r);

    cout << fact_n / (fact_r * fact_nmr) << endl;

    return 0;
}